client/src/ui/molecules/document-survey/surveys.ts

```diff
@@ -0,0 +1,25 @@
+import { Doc } from "../../../document/types";
+
+export interface Survey {
+  key: string;
+  show: (doc: Doc) => boolean;
+  rate: number;
+  src: string;
+  teaser: string;
+  question: string;
+}
+
+export const SURVEYS: Survey[] = [
+  {
+    key: "css_cascade_2022",
+    show: (doc: Doc) =>
+      /CSS/i.test(
```

Comment:
probably we want to do `/en-US\/docs\/Web\/CSS(\/|$)/i`

We only wanna show this in `en-US` and I guess we wanna restrict it to the reference.

Reply:
Important point, yes.

This would exclude [`docs/Web/API/CSS*`](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Object_Model) and also [`docs/Learn/CSS`](https://developer.mozilla.org/en-US/docs/Learn/CSS), which probably makes sense. But I'll check.

Reply:
Fixed in 67c9e6ca05180a17b83da479936e549ebb1ab462.